### ISSUE TITLE
What are the differences between `iresnet_oi_lowprop` and `iresnet_oi`?

### ISSUE THREAD

**Issue by yekeren** [alirezazareian/vspnet#7]
Dear author, I wonder about the differences between the `iresnet_oi_lowprop` and `iresnet_oi`. It seemed to me that for the weakly supervised scene graph generation (0053.ipynb), you use a much smaller proposal set (20 instead of 300), so how did you filter to get only 20 proposals per image? Also, I got confused about the following numbers:

- You mention in sec 3.1 that the size of typical $n_e$ is 300, did you try using 300 proposals in the experiments?

- At the end of Section 4.1 the paper says 200 proposals for all methods, are the ipynb files denoting your setting mentioned in your paper?

**Comment by alirezazareian** [comment]
The number of proposals is a config parameter of most region proposal methods, including Faster R-CNN which we used, and can be easily changed to try the model with different number of proposals. After non-max suppression, we sort the proposals with respect to their confidence scores, and pick the top k.

Generally, a higher number of proposals leads to higher recall, and lower speed. However, for weakly supervised setting, too many proposals make it difficult for the alignment procedure to choose the right one. In fully supervised setting, the alignment is trivially done using IoU, so increasing the number of proposals does not make the alignment more ambiguous. Hence, we used the best configuration in each setting, namely 20 proposals for weakly supervised, and 300 proposals for fully supervised.

Since the number of proposals affects inference time, we used 200 proposals in another experiment, only to calculate the inference time of the model, so we can compare to the results reported by FactorizableNet, which also used 200 proposals.
